### ISSUE TITLE
how to add openldap users to atlasdb

### ISSUE THREAD

**Issue by cha-uth** [OHDSI/Broadsea#88]
Got the develop branch running with openldap.  Everything is working as expected.

But when a openldap user login, I get a 'user1', or 'user2' displayed although the ldap user names are something else.  Also nothing can be accessed.  So how do I add them to the atlasdb?  Just need to know the related tables and schema (I assume it's webapi schema).  I will need to add at last one user with a system  level so that I can set things up.

Thanks.

**Comment by alondhe** [comment]
You'll need to use a tool like Apache Directory Studio to alter the display names of the users

**Comment by cha-uth** [comment]
I haven't tried it yet

**Comment by RomainTching** [comment]
Hi! Following along, I'm also trying to make use of the openldap solution, but I'm not very familiar with it. Could you clarify what a tool like Apache Directory Studio would be needed for and what can be done from the .env file? 
From what I understand, you can define several users and their passwords from the .env file, and as mentioned here when logging in they will be named "user1", "user2"... unless renamed from Apache Directory Studio. 
I can't login to ATLAS with the admin credentials set in the .env so I assume that may only be used from Apache Directory Studio as well?
I also saw [here ](https://github.com/OHDSI/Broadsea/issues/82#issuecomment-1563490008) that there should be a wizard to map LDAP users to ATLAS role, but I can't find it when logged as simple user, so my guess is that I need to be logged in as an admin user, but then how do I set any user as admin?

**Comment by RomainTching** [comment]
Sorry, I'm new to LDAP. What should be the Authentication Parameter credentials to use for Apache Directory Studio to connect to the openLDAP in the container? The admin credentials set in the .env? I get an authentication error so I'm not sure if I need to use one of the `uid`, `ou` or `dc` attributes...

**Comment by cha-uth** [comment]
Romain use this for the bind user

cn=ldapadmin, dc=Whatever-you-set-in your-env-file, dc=whatever-you-set-in-your-env-file

**Comment by RomainTching** [comment]
That seems to work! We were able to proceed with all these steps up to the point of assigning ATLAS roles to new users. Thanks a lot for your patience and help. 
Now this process raised another issue for source JDBC strings that I opened separately [here ](https://github.com/OHDSI/Broadsea/issues/93) since I'm not sure if it's directly related to OpenLDAP configuration or some kind of difference between the main and develop branch.
